http_client.rst

```diff
@@ -1442,6 +1455,120 @@ However, using ``MockResponse`` allows simulating chunked responses and timeouts
 
     $mockResponse = new MockResponse($body());
 
+Testing Request Data
+~~~~~~~~~~~~~~~~~~~~
+
+The examples above describe how to return desired response. What if you wanted
+to also test the request itself? ``MockResponse`` comes with helper methods:
+
+* ``getRequestMethod()`` - returns HTTP method
+* ``getRequestUrl()`` - returns URL the request would be sent to
+* ``getRequestOptions()`` - returns an array containing other information about
+  the request such as headers, query parameters, body content etc.
+
+Usage example::
+
+    $mockResponse = new MockResponse('', ['http_code' => 204]);
+    $httpClient = new MockHttpClient($mockResponse, 'https://example.com');
+
+    $response = $httpClient->request('DELETE', 'api/article/1337', [
+        'headers' => [
+            'Accept: */*',
+            'Authorization: Basic YWxhZGRpbjpvcGVuc2VzYW1l',
+        ],
+    ]);
+
+    // returns "DELETE"
+    $mockResponse->getRequestMethod();
+
+    // returns "https://example.com/api/article/1337"
+    $mockResponse->getRequestUrl();
+
+    // returns ["Accept: */*", "Authorization: Basic YWxhZGRpbjpvcGVuc2VzYW1l"]
+    $mockResponse->getRequestOptions()['headers'];
+
+Example
+~~~~~~~
+
+The following standalone example demonstrates a way to use HTTP client and
+test it in real application::
```

Comment:
```suggestion
test it in a real application:
```

Reply:
Thanks. The double `:` must stay for code block though.

Reply:
Ah, lacking the RST knowledge